<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-02-25T18:10:02Z" name="Default suite" finished-at="2019-02-25T18:10:19Z" duration-ms="17022">
    <groups>
    </groups>
    <test started-at="2019-02-25T18:10:02Z" name="Default test" finished-at="2019-02-25T18:10:19Z" duration-ms="17022">
      <class name="com.qa.tests.HomePageTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:02Z" name="setUp" finished-at="2019-02-25T12:10:06Z" duration-ms="4013" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="signInClick()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:06Z" name="signInClick" finished-at="2019-02-25T12:10:07Z" duration-ms="729" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInClick -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:07Z" name="setUp" finished-at="2019-02-25T12:10:10Z" duration-ms="3144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:07Z" name="tearDown" finished-at="2019-02-25T12:10:07Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="validateCreateList()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:10Z" name="validateCreateList" finished-at="2019-02-25T12:10:11Z" duration-ms="1251" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateCreateList -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:11Z" name="tearDown" finished-at="2019-02-25T12:10:11Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:11Z" name="setUp" finished-at="2019-02-25T12:10:14Z" duration-ms="3494" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validateSearch()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:14Z" name="validateSearch" finished-at="2019-02-25T12:10:19Z" duration-ms="4355" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Amazon.com: Nike Shoes] but found [Amazon.com: Nike Shoes: Clothing, Shoes & Jewelry]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Amazon.com: Nike Shoes] but found [Amazon.com: Nike Shoes: Clothing, Shoes & Jewelry]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.qa.tests.HomePageTest.validateSearch(HomePageTest.java:40)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateSearch -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.tests.HomePageTest@55fe41ea]" started-at="2019-02-25T12:10:19Z" name="tearDown" finished-at="2019-02-25T12:10:19Z" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.tests.HomePageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
